awsproviderlint/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go

```diff
@@ -719,7 +719,7 @@ func isReservedDataSourceFieldName(name string) bool {
 
 func isReservedResourceFieldName(name string, s *Schema) bool {
 	// Allow phasing out "id"
-	// See https://github.com/terraform-providers/terraform-provider-aws/pull/1626#issuecomment-328881415
+	// See https://github.com/hashicorp/terraform-provider-aws/pull/1626#issuecomment-328881415
```

Comment:
Vendored file should remain:

```suggestion
	// See https://github.com/terraform-providers/terraform-provider-aws/pull/1626#issuecomment-328881415
```